system-tests/tests/settings-cli-modify-bool

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+# Check that cobbler-settings can modify value from the YAML
+
+# Arrange
+source ${SYSTESTS_PRELUDE} && prepare
+
+set -x -e -o pipefail
+
+TEST_CONFIG="${tmp}/settings-bool.yaml"
+cat <<-END >$TEST_CONFIG
```

Comment:
I was just wondering why is needed here to load all the settings while in the original PR is not like this.

Reply:
The reason is that the original PR has the schema from 3.4.0 that makes all settings optional and provides defaults. The schema for 3.3.3 requires adding a lot of keys before the settings file is marked as valid.

Reply:
I understand the point now. Looks good then :)